.github/ISSUE_TEMPLATE/enhancement-request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Enhancement Request
+about: Suggest an enhancement to the KUDO project
+title: ''
+labels: kind/kep
```

Comment:
Shouldn't Kep Enhancements be handled as a PR with a new KEP object and not an Issue?

Maybe a short sentence about when a "feature request" is a sufficient and when a KEP is a required

Reply:
Here are some thoughts:

> Shouldn't Kep Enhancements be handled as a PR with a new KEP object and not an Issue?
> 

They should. However, this enhancement requests are requested by users where we cannot assume that a feature (or enhancement) is already mature enough to have a related `KEP`. Everyone submitting an enhancement request does not need to know the tiny details of the PM process. Instead, I would assume someone with more deeper project knowledge will vet these submissions and ensure to route it properly.

Also, anyone should be able to request and contribute ideas, enhancements or features without having to be the `author` of an actual `KEP`. We cannot assume that everyone with a good idea is comfortable enough using our `KEP` process directly. I think this can make it easier for users, community members or partners to interface with the project. This way we can for instance collect multiple enhancement requests and discuss them before it is articulated into an actual `KEP`. 

The flow I can picture is:

Articulation of a request as an `Enhancement Request`-issue by a user -> Discussion in the issue between the user, community members and OWNERs -> Creation of the actual KEP by the issue-author, another community member that sees fit or an OWNER who vets for it.

> Maybe a short sentence about when a "feature request" is a sufficient and when a KEP is a required

Agreed, I also don't wanna make it a too fuzzy process. Imo saying `Suggest an enhancement to the KUDO project` with `What would you like to be added` and `Why is this needed` should be sufficient to get a conversation started.

Why I don't wanna use the term `feature request`: We are talking specifically of a `KUDO enhancement proposal` not `KUDO feature proposal` and I also see in the wider k8s community the notion to step away from the word `feature` and replace it with `enhancement` ( ref https://github.com/kubernetes/sig-release/pull/298#issuecomment-422481846 )

What do you think?

Instead of adding the label ~~`kind/kep`~~ we should add the label `kind/enhancement` so it will be more an enhancement-sort for `kind/kep`.